simple/schema.py

```diff
@@ -111,203 +142,330 @@ class Gravity(enum.Enum):
     __tablename__ = 'ProperMotions'
-    source = Column(String(100), ForeignKey('Sources.source', ondelete='cascade', onupdate='cascade'),
-                    nullable=False, primary_key=True)
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
     mu_ra = Column(Float, nullable=False)
     mu_ra_error = Column(Float)
     mu_dec = Column(Float, nullable=False)
     mu_dec_error = Column(Float)
-    adopted = Column(Boolean)  # flag for indicating if this is the adopted measurement or not
+    adopted = Column(Boolean)
     comments = Column(String(1000))
-    reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'), primary_key=True)
+    reference = Column(
+        String(30),
+        ForeignKey("Publications.reference", onupdate="cascade"),
+        primary_key=True,
+    )
 
 
 class RadialVelocities(Base):
     # Table to store radial velocities, in km/sec
     __tablename__ = 'RadialVelocities'
-    source = Column(String(100), ForeignKey('Sources.source', ondelete='cascade', onupdate='cascade'),
-                    nullable=False, primary_key=True)
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
     radial_velocity = Column(Float, nullable=False)
     radial_velocity_error = Column(Float)
-    adopted = Column(Boolean)  # flag for indicating if this is the adopted measurement or not
+    adopted = Column(Boolean)
     comments = Column(String(1000))
-    reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'), primary_key=True)
+    reference = Column(
+        String(30),
+        ForeignKey("Publications.reference", onupdate="cascade"),
+        primary_key=True,
+    )
 
 
 class SpectralTypes(Base):
     # Table to store spectral types, as strings
     __tablename__ = 'SpectralTypes'
-    source = Column(String(100), ForeignKey('Sources.source', ondelete='cascade', onupdate='cascade'),
-                    nullable=False, primary_key=True)
+    source = Column(
+        String(100),
+        ForeignKey("Sources.source", ondelete="cascade", onupdate="cascade"),
+        nullable=False,
+        primary_key=True,
+    )
     spectral_type_string = Column(String(10), nullable=False)
     spectral_type_code = Column(Float, nullable=False)
     spectral_type_error = Column(Float)
     regime = Column(Enum(Regime, create_constraint=True, native_enum=False),
                     primary_key=True)  # restricts to a few values: Optical, Infrared
-    adopted = Column(Boolean)  # flag for indicating if this is the adopted measurement or not
+    # regime = Column(
+    #    String(30), ForeignKey("Regimes.regime", onupdate="cascade"), primary_key=True
+    # )
+    adopted = Column(Boolean)
```

Comment:
If you wanted to bring it closer as part of this PR, you could consider setting _regime_ as just a `Column(String(30), primary_key=True)` and add the `ForeignKey` constraint later